api/events/schema.py

```diff
@@ -74,7 +74,8 @@ def mutate(self, info, **kwargs):
                 end_time=kwargs['end_time'],
                 number_of_participants=kwargs['number_of_participants'],
                 checked_in=False,
-                cancelled=True)
+                cancelled=True,
```

Comment:
When we run sync event data, events with `cancelled=True` will be archived and so will events with `auto_cancelled=True` for that reason. Will this allow us to keep track of the number of auto_cancelled events?

Reply:
Events that get archived during sync are the ones that were initially in our db and then get canceled from the google calendar. Therefore all events canceled from the google calendar are archived in our db. If the event was not canceled from the google calendar it will not get archived in our db.

Reply:
Since auto-cancellation does not happen from google calendar, such events will not get archived during sync.